collectors/iscsi.py

```diff
@@ -92,6 +97,17 @@ class ISCSIGateway(BaseCollector):
     def __init__(self, *args, **kwargs):
         BaseCollector.__init__(self, *args, **kwargs)
 
+        # Since the module can be imported by a parent class but not
+        # instantiated, the rtslib import is deferred until the first instance
+        # of the the class is created. This keeps the parent module simple
+        # and more importantly generic
+        if 'rtslib_fb.root' not in sys.modules.keys():
+
+            try:
+                import rtslib_fb.root as RTSRoot
+            except ImportError:
+                raise ImportError("rtslib_fb python package is missing")
```

Comment:
I'd prefer the code actually acknowledged the scenario of no rtslib_fb especially since on import you won't see a dependency problem - until you instantiate.